Draft for the May's Newsletter in the filename: newsletter_14_May2020.md (#1108)
alan-turing-institute/the-turing-way: communications/newsletters/newsletter_14_May2020.md

# Attend this week’s Collaboration Café, learn about Mental Health Awareness Week and find other updates!

Hello Turing Way friends!

This month as we continue to adjust to the current norm, we wish you good health and hope that you and your loved ones are keeping well. 
It’s not easy keeping morale up with the current state of the pandemic, therefore, in this newsletter, we talk about the importance of self-care and highlight the Mental Health Foundation who are hosting [Mental Health Awareness Week](https://www.awarenessdays.com/awareness-days-calendar/mental-health-awareness-week-2020/) from 18 to 24 May 2020. 🌻

We congratulate Kirstie Whitaker, the project lead of _The Turing Way_ for starting her new role as the [lead of Tools, Practices and Systems research programme](https://www.turing.ac.uk/news/dr-kirstie-whitaker-named-institutes-programme-lead-open-source-tools-practices-and-systems) at The Alan Turing Institute, London. 🎊

> 🗓 Join us at our upcoming Collaboration Café this week on 20 May 2020 at 19:00 ([check your local time](https://arewemeetingyet.com/London/2020-05-20/19:00/TuringWay-CollaborationCafe)). 

Reviewer: ```suggestion
🗓 Join us at our upcoming Collaboration Café this week on 20 May 2020 at 19:00 BST ([check your local time](https://arewemeetingyet.com/London/2020-05-20/19:00/TuringWay-CollaborationCafe)). 
```

More details are available on [this HackMD](https://hackmd.io/@KirstieJane/CollabCafe).

> 📢 [International Neuroinformatics Coordinating Facility (INCF)](https://www.incf.org/blog/call-mentors-join-us-gsod-2020) is participating at the [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants).

Reviewer: ```suggestion
📢 [International Neuroinformatics Coordinating Facility (INCF)](https://www.incf.org/blog/call-mentors-join-us-gsod-2020) is participating in the [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants).
```

Kirstie joined the institute in 2017 as a Turing Research Fellow and was awarded a seed funding grant to "make reproducible research too easy not to do" alongside the Turing’s Director of Research Engineering Martin O'Reilly.
In her own words: 
*"I've always loved working at the Turing, the breadth of skills and interests has always meant fascinating discussions in the (now virtual) kitchen over coffee. 
I'm delighted that I have the opportunity to support our national responsibility to maintain the open source tools, practices and systems that underpin and enable the ethical and responsible use of data to solve the biggest challenges around the world."* 
Read more in [this announcement](https://www.turing.ac.uk/news/dr-kirstie-whitaker-named-institutes-programme-lead-open-source-tools-practices-and-systems).

![Tweet about Kirstie's new position](images/kw-May2020.png)

*Tweet by [The Turing Institute](https://twitter.com/turinginst/status/1255054249807024128?s=20)*

## Upcoming workshops and events

### Boost Research Reproducibility with Binder Workshop

> In collaboration with [UK DRI](https://ukdri.ac.uk/), _The Turing Way_ is hosting a half-day workshop on Binder on 11 June 2020, from 13:00 to 17:00.

Reviewer: ```suggestion
In collaboration with [UK DRI](https://ukdri.ac.uk/), _The Turing Way_ is hosting a half-day workshop on Binder on 11 June 2020, from 13:00 to 17:00 BST.
```

Binder allows you to share your software from GitHub in a single clickable link, making it easier for users to not worry about the computational environments, dependencies and platforms.
In this workshop, we will discuss reproducible computing environments, show examples of others’ projects in myBinder.org and help you learn how to prepare a binder-ready project. 
At the end of the workshop, you will be able to take some of your own content (in Jupyter notebook or scripts) and prepare it so that it can be used by others on myBinder.org.
More information is available on the [registration page online](https://www.eventbrite.co.uk/e/workshop-boost-research-reproducibility-with-binder-registration-105228216428).

> If you have previous experience with Binder and would like to join as a helper, please reach out to us by emailing theturingway@gmail.com.

Reviewer: ```suggestion
If you have previous experience with Binder and would like to join as a helper, please reach out to us by emailing [theturingway@gmail.com](mailto:theturingway@gmail.com).
```

### Google Seasons of Docs

> INCF is taking part in this year's [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants) as an open source organisation and _The Turing Way_ is listed as one of its projects.

Reviewer: ```suggestion
INCF is taking part in this year's [Google Seasons of Docs](https://developers.google.com/season-of-docs/docs/participants) (GSOD) as an open source organisation and _The Turing Way_ is listed as one of its projects.
```

Introducing the acronym you use below

During its exploration phase from May 11 to June 8, we invite the professional technical writers to connect with us and discuss the project ideas that can allow them to bring in their perspectives and unique skill set.
_The Turing Way_ involves a large community of volunteers who edit, review and maintain its resources online on GitHub.
As a book project, we can benefit tremendously from the support of experienced individuals who can provide editorial support and related guidance in our project.
We have compiled some recommendations that can be used as the starting points for the GSOD candidates.
Please find details on [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/1075), and feel free to comment there if any further information will be useful.

### Data Feminism Book and “Challenging Power in Data Science” seminar 

[Data Feminism](https://mitpress.mit.edu/books/data-feminism) is a recent book by Catherine D'Ignazio and Lauren F. Klein that explains the concept of data feminism as a way of thinking about data science and its communication informed by decades of intersectional feminist activism and critical thought. 

> They will be giving a talk hosted by The Alan Turing Institute to discuss how challenges to the male/female binary can challenge other hierarchical classification systems; how an understanding of emotion can expand our ideas about effective data visualization; and how the concept of “invisible labor” can expose the significant human efforts required by our automated systems. 

Reviewer: You use challenges twice in this sentence, is there another word we could use?

Author: ```suggestion
They will be giving a talk hosted by The Alan Turing Institute to discuss how challenging the male/female binary system can help investigate other hierarchical classification systems; how an understanding of emotion can expand our ideas about effective data visualization; and how the concept of “invisible labor” can expose the significant human efforts required by our automated systems. 
```

Author: I took that from the eventbrite description, rephrased that now.

### ReproHack 

Anna Krystalli hosted an online ReproHack event on 14 May 2020, where Sarah Gibson delivered a talk on Binder.
Both members are active contributors to _The Turing Way_ and fellows of the Software Sustainability Institute.
[ReproHacks](https://github.com/reprohack) are one-day workshop events, where participants attempt to reproduce published research of their choice from a list of proposed papers with publicly available associated code and data.
See [the workshop page](https://n8cir.org.uk/events/remote-reprohack/) for details. 

## Tips & Tricks for new contributors

Are you new to _The Turing Way_ and would like to learn more about the project? Listen to the insightful talk delivered by the project lead, Kirstie Whitaker, at [OpenMR Benelux 2020](https://openmrbenelux.github.io/products/product08/) recording from which is available [online on YouTube](https://www.youtube.com/watch?v=dlycvMU45ek&feature=youtu.be).

### Contributor in focus: Camila Rangel Smith

Camila Rangel Smith is a Research Data Scientist at The Alan Turing Institute.

> She is also a co-founder of the [CEVALE2VE project](http://www.cevale2ve.org/en/home/) that has become consolidated into [LA-CoNGA physics](http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.

Reviewer 2: Could we put a capital P in Physics? Its my fault that i didn't write it correctly on the email :)

Author: ```suggestion
She is also a co-founder of the [CEVALE2VE project](http://www.cevale2ve.org/en/home/) that has become consolidated into [LA-CoNGA Physics](http://laconga.redclara.net/), an EU Erasmus+ funded project with a mission to create a Latin American and European Community for Advanced Physics.
```

> She is translating _The Turing Way_ resources into Spanish in collaboration with her friends and colleagues from Venezuela and Mexico. 

Reviewer 2: I think the contributors are from Venezuela and Spain, I'm not aware of anyone in Mexico at the moment. :)

Author: Ah! Thanks for pointing that. I am correcting that right away.

Author: Ah! Thanks for pointing that. I am correcting that right away.

In her words: *“I think The Turing Way is an excellent resource that can change the way science is done (I wish I had it when I started my PhD). 

> Although the international language of science is English, I know for a fact that not everyone in places like Latin-American has the time and resources to learn it, so I think we must do everything we can to break those barriers and improve the accessibility of knowledge for everyone. 

Reviewer 2: My fault again, I think is "Latin-America" without the "n" at the end.

Author: ```suggestion
Although the international language of science is English, I know for a fact that not everyone in places like Latin-America has the time and resources to learn it, so I think we must do everything we can to break those barriers and improve the accessibility of knowledge for everyone. 
```

We, in fact, encourage you to take a break whenever needed and return to your project ideas in your own time.

### Staying connected with others

To avoid feeling disconnected while you work from home, you can join others over online coffee breaks or co-working calls. 
These are short informal calls that can help you stay connected with others in your workplace and social network, create opportunities to exchange ideas, stay motivated and show kindness towards each other. 
Kirstie has shared her thoughts and ideas on running effective virtual coffee calls on [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/965).

### Share your tips on self-care

We would love to hear your tips and recommendations on self-care and ensuring the well being of your team members. Please comment under [this issue](https://github.com/alan-turing-institute/the-turing-way/issues/965) on the GitHub repository.
Your recommendations will be included to the chapter on remote collaboration ([see details](https://github.com/alan-turing-institute/the-turing-way/pull/962)).

> Have suggestions to improve accessibility in our community? Reach out to us by emailing theturingway@gmail.com.

Reviewer: ```suggestion
Have suggestions to improve accessibility in our community? Reach out to us by emailing [theturingway@gmail.com](mailto:theturingway@gmail.com).
```
